src/index.css

```diff
@@ -2,3 +2,371 @@
   min-height: 100vh;
   height: 100vh;
 }
+
+
+head,
+body {
+  background-color: #ecf0f5;
+}
+.content-wrapper {
+  min-height: initial !important;
+}
+.bg-auth {
```

Comment:
Hey @dallen72, thanks for letting us know! I'm going to close out this PR since there's still some work to be done on this and see if with merging in #70 we can get the log in form completed easier. :)